### ISSUE TITLE
Remove unnecessary contains() check

### ISSUE THREAD

**Issue by berry120** [quelea-projection/Quelea#435]
LivePanel, lines 430-432 contains an unnecessary `contains()` check - we can just call `remove()` as-is, and it will only remove the item if it exists.

**Comment by varnaa** [comment]
@berry120 Hi, I'd like to work on this issue.

**Comment by vishal675** [comment]
@berry120  Hi, can you assignee this issue to me

**Comment by berry120** [comment]
@varnaa Afraid we're limiting the good first issues to one per contributor at the mo, just to make sure as many people as possible can contribute.

@vishal675 All yours!

**Comment by varnaa** [comment]
> @varnaa Afraid we're limiting the good first issues to one per contributor at the mo, just to make sure as many people as possible can contribute.
> 

@berry120  Yes, i totally agree. I commented on multiple issues hoping to be assigned to at least one xD. Thanks for making sure that everyone gets equal opportunity :)

**Comment by berry120** [comment]
@vishal675 Are you still planning to work on this? If we could have a PR in the next couple of days that'd be great, otherwise we'll need to open it back up in the interest of fairness. If you need any help just shout 👍
